src/game/ZONES.ts

```diff
@@ -33,164 +33,29 @@ interface Zone {
-      },
-      {
-        id: 2412,
-        name: 'The Council of Blood',
-      },
-      {
-        id: 2399,
-        name: 'Sludgefist',
-      },
-      {
-        id: 2417,
-        name: 'Stone Legion Generals',
-      },
-      {
-        id: 2407,
-        name: 'Sire Denathrius',
-      },
-    ],
-    brackets: {
-      min: 173,
-      max: 230,
-      bucket: 3,
-      type: 'Item Level',
-    },
-    partitions: [
-      {
-        name: '9.0',
-        compact: '9.0',
-      },
-      {
-        name: '9.0.5',
-        compact: '9.0.5',
-        default: true,
-      },
-      {
-        name: '9.1',
-        compact: '9.1',
-      },
-    ],
-  },
-  {
-    id: 28,
-    name: 'Sanctum of Domination',
-    frozen: false,
-    encounters: [
-      {
-        id: 2423,
-        name: 'The Tarragrue',
-      },
-      {
-        id: 2433,
-        name: 'The Eye of the Jailer',
-      },
-      {
-        id: 2429,
-        name: 'The Nine',
-      },
-      {
-        id: 2432,
-        name: "Remnant of Ner'zhul",
-      },
-      {
-        id: 2434,
-        name: 'Soulrender Dormazain',
-      },
-      {
-        id: 2430,
-        name: 'Painsmith Raznal',
-      },
-      {
-        id: 2436,
-        name: 'Guardian of the First Ones',
-      },
-      {
-        id: 2431,
-        name: 'Fatescribe Roh-Kalo',
-      },
-      {
-        id: 2422,
-        name: "Kel'Thuzad",
-      },
-      {
-        id: 2435,
-        name: 'Sylvanas Windrunner',
-      },
-    ],
-    brackets: {
-      min: 203,
-      max: 260,
-      bucket: 3,
-      type: 'Item Level',
-    },
-    partitions: [
-      {
-        name: '9.1',
-        compact: '9.1',
-      },
-      {
-        name: '9.1.5',
-        compact: '9.1.5',
-        default: true,
-      },
-    ],
-  },
-  {
-    id: 29,
-    name: 'Sepulcher of the First Ones',
+    id: 31,
+    name: 'Vault of the Incarnates',
     frozen: false,
     encounters: [
-      sepulcher.bosses.VigilantGuardian,
-      sepulcher.bosses.Dausegne,
-      sepulcher.bosses.Xymox,
-      sepulcher.bosses.PrototypePantheon,
-      sepulcher.bosses.Skolex,
-      sepulcher.bosses.Halondrus,
-      sepulcher.bosses.Lihuvim,
-      sepulcher.bosses.Anduin,
-      sepulcher.bosses.LordsOfDread,
-      sepulcher.bosses.Rygelon,
-      sepulcher.bosses.Jailer,
+      vaultOfTheIncarnates.bosses.Eranog,
+      vaultOfTheIncarnates.bosses.Terros,
+      vaultOfTheIncarnates.bosses.PrimalCouncil,
+      vaultOfTheIncarnates.bosses.Sennarth,
+      vaultOfTheIncarnates.bosses.Dathea,
+      vaultOfTheIncarnates.bosses.KurogGrimtotem,
+      vaultOfTheIncarnates.bosses.BroodkeeperDiurna,
+      vaultOfTheIncarnates.bosses.Raszageth,
     ],
     brackets: {
-      min: 236,
-      max: 280,
+      min: 376,
+      max: 424,
       bucket: 3,
       type: 'Item Level',
     },
     partitions: [
       {
-        name: '9.2',
-        compact: '9.2',
+        name: '10.0',
+        compact: '10.0',
```

Comment:
Looking at castle nathria, we might need to include 10.0.5 as well to the partitions? Not positive but worth checking

Reply:
The partitions should match what WCL has. We'll likely add a 10.0.5 partition due to class changes, but there's not a need to pre-emptively add them.